static/src/javascripts/projects/journalism/views/audioSeriesContributions.html

```diff
@@ -1,11 +1,16 @@
 <section class="flagship-audio__contributions-banner fc-container">
     <div class="fc-container__inner">
         <div class="fc-container__header"> </div>
-        <div class="fc-container--rolled-up-hide fc-container__body">
-            <h2>Support The Guardian</h2>
-            <p>Help us deliver the independent
-                <br> journalism the world needs</p>
-            <a class="contributions__contribute contributions__option-button" data-link-name="audio-series-contributions">Support the Guardian</a>
+        <div class="fc-container--rolled-up-hide fc-container__body flagship-audio__contributions-banner-holder">
+            <div class="flagship-audio__contributions-banner-text">
+                <h2>Support The Guardian</h2>
+                <p>Help us deliver the independent
+                    <br> journalism the world needs</p>
+                <a class="contributions__contribute contributions__option-button" data-link-name="audio-series-contributions">Support the Guardian</a>
+            </div>
+            <div class="flagship-audio__contributions-banner-image">
+                <svg xmlns="http://www.w3.org/2000/svg" width="300" height="107"><path d="M0 83.118h12.148v23.824H0zm16.901-36h12.148v59.824H16.901zm16.902-21.177h12.148v81H33.803zm16.901 21.177h12.148v59.824H50.704zM67.606 18h12.148v88.941H67.606zm16.901 41.824h12.148v47.118H84.507zm16.901-12.706h12.148v59.824h-12.148zm16.902-8.471h12.148v68.294H118.31zM135.211 0h12.148v106.941h-12.148zm16.902 49.765h12.148v57.176h-12.148zm16.901-28.059h12.148v85.235h-12.148zm16.901 8.47h12.148v76.765h-12.148zM202.817 0h12.148v106.941h-12.148zm16.901 47.118h12.148v59.824h-12.148zm16.902-21.177h12.148v81H236.62zm16.901-17.47h12.148v98.471h-12.148zm16.902 41.294h12.148v57.176h-12.148zm16.901 33.882h12.148v23.294h-12.148z" fill="#FF6A56"/></svg>
```

Comment:
If it is an SVG we may reuse somewhere else, perhaps move it into `static/src/inline-svgs/journalism/audio`?

Reply:
Yes, fair enough. So I could use an image tag for that? `<img src="image.svg" />`

Reply:
In fact, I will leave this as a embedded SVG. Connecting this lodash template up with the correct external file gets fiddly.